Bio/Phylo/_utils.py

```diff
@@ -640,3 +640,195 @@ def draw_clade(clade, x_start, color, lw):
 
     if do_show:
         plt.show()
+
+
+def build_ete3tree(tree, ete_root):
+    """Convert a Tree object to a ete3's Tree."""
+    for node in tree.root:
+        ete_node = ete_root.add_child(name=node.name, dist=node.branch_length)
+        ete_node.add_features(branch_length=node.branch_length)
+
+        if hasattr(node, "confidences"):
+            if len(node.confidences) == 1:
+                confidence = node.confidences[0].value
+                ete_node.add_features(confidence=confidence, support=confidence)
+            else:
+                ete_node.add_features(confidences=node.confidences)
+
+        if hasattr(node, "taxonomy"):
+            ete_node.add_features(taxonomy=node.taxonomy)
+
+        build_ete3tree(node, ete_node)
+
+
+def draw(
```

Comment:
How about this approach to reduce the import-based code branching:
1. Split this function into `draw_ete3` and `draw_matplotlib`, where each function throws an error if the necessary imports aren't available and otherwise proceeds smoothly. (Factor out common code into helper functions if that improves the flow.)
2. At the bottom of this module, define `draw` at import time as an alias depending on whether `ete3` is a available:
```
try:
    import ete3
    draw = draw_ete3
except ImportError:
    draw = draw_matplotlib
```

Reply:
I haven't had time to refactor today, but that's a good idea.